### PR TITLE
[CODEOWNERS] Remove invalid code owners

### PR DESCRIPTION
**Pull request by liurunliang**
This PR is to remove invalid github accounts from `codeowners` file. Also remove the service entry if having empty owner.

### CODE REVIEW

File: CODEOWNERS [hunk 1]
```diff
@@ -108,144 +108,144 @@
...
+/specification/intune/
 
 # PRLabel: %IotHub
 /specification/iothub/ @rkmanda
 
 # PRLabel: %KeyVault
-/specification/keyvault/ @heaths @randallilama @jlichwa
+/specification/keyvault/ @heaths @jlichwa
 
 % PRLabel: %Load Test Service
```
Comment (raych1) [inline CODEOWNERS:127]: First '%' should be '#'.

**Comment by raych1** [comment]
+@weshaggard , will empty owner cause issues for dependencies of CODEOWNER?

File: CODEOWNERS [hunk 2]
```diff
@@ -6,82 +6,82 @@
 #########
 
 # PRLabel: %Cognitive Services
-/dev/cognitiveservices/data-plane/Language/ @assafi @rokulka @ChongTang @annatisch @heaths @Azure/api-stewardship-board
+/dev/cognitiveservices/data-plane/Language/ @assafi @ChongTang @annatisch @heaths @Azure/api-stewardship-board
 
-/specification/adp/ @Azure/adp
+/specification/adp/
```
Comment (mikekistler) [inline CODEOWNERS:11]: It was my understanding that the automation needs at least one id following the directory name to work correctly, so removing the last id for a directory seems problematic.

File: CODEOWNERS [hunk 3]
```diff
@@ -6,82 +6,82 @@
 #########
 
 # PRLabel: %Cognitive Services
-/dev/cognitiveservices/data-plane/Language/ @assafi @rokulka @ChongTang @annatisch @heaths @Azure/api-stewardship-board
+/dev/cognitiveservices/data-plane/Language/ @assafi @ChongTang @annatisch @heaths @Azure/api-stewardship-board
```
Comment (mikekistler) [inline CODEOWNERS:10]: Shouldn't there be a "PRLabel" line above this directory?

**Review by mikekistler — CHANGES_REQUESTED** [review]
I think we need a solution for directories where there are no known code owners.  We want to keep the directories so the PR labeling still works.

**Comment by weshaggard** [comment]
We should definitely avoid having empty codeowners I'm not sure how this edit was made. How were the invalid owners determined? Did you only look at the squiggly lines on github? If so that doesn't mean they are not necessarily the right owner it might also mean that they just don't have write permission to the repo. See:
image

We should avoid blindly removing folks and only remove them if we know they are no longer accurate. @josefree any thoughts on how to identify correct owners?

**Review by raych1 — APPROVED** [review]

**Comment by raych1** [comment]
Confirmed by @weshaggard , empty owner won't block it. As synced offline, we removed those alias that are invalid and keep others that are valid but without write permission. For empty lines, we removed that service entry from file and sent you in email for tracking.

**Review by mikekistler — APPROVED** [review]
Looks good! 👍

**Comment by weshaggard** [comment]
Thanks for the clean-up thus far but lets also try to remove any owners that aren't part of the Azure org on github.

**Review by raych1 — APPROVED** [review]

**Review by weshaggard — APPROVED** [review]

**Pull request merged**